app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person["name"], person["age"]) for person in people]
+    for person in people:
+        if "wife" in person and person["wife"]:
+            husband = Person.people[person["name"]]
+            husband.wife = Person.people[person["wife"]]
+        elif "husband" in person and person["husband"]:
+            wife = Person.people[person["name"]]
+            wife.husband = Person.people[person["husband"]]
```

Comment:
`Fix everywhere (besides conditions, of course)` -- you've missed the last part 🤫
You should keep method get in conditions